ios/FluentUI.Demo/FluentUI.Demo/Demos/TableViewHeaderFooterViewDemoController.swift

```diff
@@ -57,7 +57,7 @@ class TableViewHeaderFooterViewDemoController: DemoController {
         tableView.register(TableViewHeaderFooterView.self, forHeaderFooterViewReuseIdentifier: TableViewHeaderFooterView.identifier)
         tableView.dataSource = self
         tableView.delegate = self
-        tableView.backgroundColor = TableViewCell.tableBackgroundColor
+        tableView.backgroundColor = UIColor(dynamicColor: view.fluentTheme.aliasTokens.colors[.background2])
```

Comment:
Feels like this value should be pulled from the table component tokens instead of directly from `AliasTokens`, to guarantee it stays coherent going forward.

Reply:
Same with the `cell.backgroundColor` below.

Reply:
It's actually just a different, custom value to match the TableViewHeaderFooterView color! Using the table component tokens would not necessarily ensure things would stay consistent. This could be updated when TableViewHeaderFooterView becomes tokenized though.